pkg/cli/image/imagesource/file.go

```diff
@@ -239,7 +239,7 @@ func (s *fileManifestService) Put(ctx context.Context, manifest distribution.Man
 	for _, option := range options {
 		if opt, ok := option.(distribution.WithTagOption); ok {
 			tagPath := filepath.Join(s.r.basePath, "v2", s.r.repoPath, "manifests", opt.Tag)
-			if err := atomicSymlink(tagPath, manifestPath); err != nil {
+			if err := atomicSymlink(tagPath, dgst.String()); err != nil {
```

Comment:
I'll defer to @smarterclayton who wrote the original bit

Reply:
@smarterclayton appreciate any thoughts

Reply:
This *seems* correct, but would need to test it.  The manifest itself should be a hardline, so if this is local that's fine.